racket/recursivitate/schelet/recursive-images.rkt

```diff
@@ -10976,72 +11057,72 @@
 0 0 24 3 1 #")"
 0 0 24 29 1 #"\n"
 0 0 17 3 71
-(109
+(117
  #";; Implementa\xC8\x9Bi sortarea prin interclasare (merge sort) pentru"
  #" imagini"
 ) 0 0 24 29 1 #"\n"
 0 0 17 3 78
-(110
+(118
  #";; Sortarea trebuie s\xC4\x83 fie \xC3\xAEn ordine descresc\xC4\x83to"
  #"are, cu func\xC8\x9Bia de ordine:"
 ) 0 0 24 29 1 #"\n"
 0 0 17 3 58
 #";; img1 < img2 = (image-height img1) < (image-height img2)"
 0 0 24 29 1 #"\n"
 0 0 17 3 103
-(111
+(119
  #";; \xC3\x8En cazul \xC3\xAEn care \xC3\xAEn\xC4\x83l\xC8\x9Bimile ima"
  #"ginilor sunt egale, se va compara l\xC4\x83\xC8\x9Bimea folosind imag"
  #"e-width"
 ) 0 0 24 29 1 #"\n"
 0 0 17 3 20 #";; Axiome mergesort:"
 0 0 24 29 1 #"\n"
 0 0 17 3 67
-(112
+(120
  #";; mergesort [x] = [x]  - o list\xC4\x83 cu un element este deja sort"
  #"at\xC4\x83"
 ) 0 0 24 29 1 #"\n"
 0 0 17 3 101
-(113
+(121
  #";; mergesort L = merge ;; mergesort L = merge (mergesort (fst-half L)"
```

Comment:
Aici e un mic typo?

Reply:
In DrRacket nu îmi apare nimic relevant.